src/test/java/teammates/test/pageobjects/InstructorFeedbackSessionsPage.java

```diff
@@ -336,7 +336,8 @@ public void fillTimeValueForDatePickerTest(String timeId, Calendar newValue) {
         WebElement dateInputElement = browser.driver.findElement(By.id(timeId));
         click(dateInputElement);
         dateInputElement.clear();
-        dateInputElement.sendKeys(TimeHelper.formatDateForSessionsForm(newValue.getTime()));
+        dateInputElement.sendKeys(TimeHelper.formatDateForSessionsForm(
+                TimeHelper.convertDateToLocalDateTime(newValue.getTime())));
```

Comment:
Same here - the removal of `TimeHelper#formatDateForSessionsForm(Date)` can wait until the tests involving `InstructorFeedbackSessionsPage` have been refactored to use `java.time`.